mycli/main.py

```diff
@@ -793,9 +793,10 @@ def format_output(self, title, cur, headers, expanded=False,
             rows = list(cur)
             formatted = self.formatter.format_output(
                 rows, headers, format_name='vertical' if expanded else None)
+            first_line = formatted[:formatted.find('\n')]
```

Comment:
@j-bennet Yeah, I think so. Plus, @meeuw is trying to get all this to work with generators, in which case we don't want to loop through the whole output.